test/WebSites/MvcTagHelpersWebSite/wwwroot/site.js

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
Fille in with document.write's or somethign so we can see expected output in browser. (same with site2.js)